.github/workflows/ci_cron_weekly.yml

```diff
@@ -178,6 +179,8 @@ jobs:
 
     runs-on: linux-arm64
     name: Python 3.12 (arm64)
+    # keep condition in sync with test_more_architectures
+    if: (github.repository == 'astropy/astropy' && (github.event_name == 'schedule' || github.event_name == 'push' || github.event_name == 'workflow_dispatch' || contains(github.event.pull_request.labels.*.name, 'Extra CI')))
```

Comment:
One idea to keep them in sync is to first run the arm64 one and to then make the other architectures conditional on the arm64 one, which would also avoid wasting CI time if the arm64 one doesn't work

Reply:
That's a good idea, but I still suggest merging this first to get the backport branch in sync.